packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -70,6 +70,28 @@ export function executeHandlebarsTemplate(
   templateString: string,
   data: Record<string, any> = {}
 ): string {
+  Handlebars.registerHelper('ifCond', function (v1, operator, v2, options) {
```

Comment:
```
Error: @opencrvs/client: src/views/PrintCertificate/PDFUtils.ts(76,39): error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
```
compiler failing here. Verify that `yarn test:compilation` is passing in `packages/client` after you fix this